twisted/web/_http2.py

```diff
@@ -579,6 +579,9 @@ def _handleWindowUpdate(self, event):
         if streamID:
             # Update applies only to a specific stream. If we don't have the
             # stream, that's ok: just ignore it.
+            if streamID not in self.streams:
+                return
```

Comment:
maybe would help to get a comment here.. like
```
# Stream no longer registered for flow control.
```

I better way would be to have

```
if self.isStreamRegistered(streamID):
   return

or 

if self.isStreamActive(streamID):
   return
```

please note that I have no idea about the internal architecture and design of HTTP2 ...so my suggestions might be wrong

the important think is that the current `if streamID not in self.streams:` is very generic and I don't think that it expresses what is going on there